src/helpers/environment.sh

```diff
@@ -16,6 +17,17 @@ function ensure_git_repository {
 }
 
 
+function gittown_dev_version {
+  local base_dir=$( dirname "${BASH_SOURCE[0]}" )
+  local git_hash=$(git -C "$base_dir" rev-parse --short HEAD)
+  local git_date=$(git -C "$base_dir" --no-pager show -s --format=%cD HEAD | cut -d ' ' -f2-4)
+
+  if ! git -C "$base_dir" remote -v > /dev/null 2>&1 | grep "https://github.com/Homebrew/homebrew.git (fetch)"; then
```

Comment:
I suggest against that. Instead you can have grep output the count of matches. This is what we have been doing throughout the repository. 
```sh
if [ "$(git -C "$base_dir" remote -v | grep -c "https://github.com/Homebrew/homebrew.git (fetch)")" == 0 ]; then
  # ...
fi
```